.github/workflows/terraform-build-test.yaml

```
@@ -0,0 +1,47 @@
+---
+name: terraform-build-test
+
+on:
+  schedule:
+    - cron: '0 0,12 * * *'
```

Consider changing the build hours to something that will happen when more people across the team are online

```suggestion
    - cron: '0 7,16 * * *'
```